### DISCUSSION TITLE
Can I run backup command on live production db?

### DISCUSSION THREAD

**Discussion by siarheipashkevich** [spatie/laravel-backup#1543] [Q&A]
Hi all.

Can I run backup command on live production single db?

**Comment by erikn69** [comment]
If it will not be used for that, what use would this package be?

**Reply by fredsal** [reply]
Mark this as answered

**Comment by siarheipashkevich** [comment]
@erikn69 sorry for the delay response.

I mean if I run backup in live (when many users active) will it affect to the data in backup and whether it will interfere with active users to work with the site.

At the moment I backup daily at 04:30 AM and backup process is ~30min.

Can I back up at any time or should I wait for the least activity?

@fredsal
